substrate/frame/revive/src/benchmarking/mod.rs

```diff
@@ -886,6 +887,29 @@ mod benchmarks {
 		assert_eq!(U256::from_little_endian(&memory[..]), runtime.ext().block_number());
 	}
 
+	#[benchmark(pov_mode = Measured)]
+	fn seal_block_author() {
+		build_runtime!(runtime, memory: [[123u8; 20], ]);
+
+		let mut digest = Digest::default();
+		digest.push(DigestItem::PreRuntime([1, 2, 3, 4], 123.encode()));
+		digest.push(DigestItem::Seal([1, 2, 4, 4], 123.encode()));
```

Comment:
Is this a realistic amount of messages to be expected in this Vec? Is this the worst case?

Reply:
I see `#[pallet::unbounded]` over `Digest` and `log` is just `alloc::vec::Vec`. I don't see anything indicating this to be unbounded.